### PR TITLE
Added Landing Page for Ecommerce website

### PR DESCRIPTION
**Pull request by Anurag021**
## Added Files:
     * nav.css
     * footer.css
     * landing.css
     * category.css
     * index.html 
     
## Preview link: https://mystifying-beaver-adc018.netlify.app/

### CODE REVIEW

File: Components/category.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.category-cards{
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
+.category-container h2{
+    text-align: center;
+    margin: 0;
```
Comment (MayankKumar10) [inline Components/category.css]: define margin 0 in reset CSS child elements inherits from it.

File: Components/landingPage.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
+.main-slider-container{
+    background-image:url('/Assests/Book-Landing-Image.png');
+    width:100%;
+    height:25rem;
+    background-repeat: no-repeat;
+    margin-top: 2rem;
+    display: flex;
```
Comment (MayankKumar10) [inline Components/landingPage.css]: write helper class for flex properties so you don't have to write everytime flex properties

File: Components/landingPage.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+.slider-icon{
+    font-size: 4rem;
+    cursor: pointer;
+    color: var(--color-gray-600)
+}
+.slider-icon:hover{
+    background-color: var(--color-gray-200);
+}
+.slider-content{
+    margin-right: 40rem;
```
Comment (MayankKumar10) [inline Components/landingPage.css:20]: don't use large margin values it will affect later when you integrate more components into your project

**Review by MayankKumar10 — APPROVED** [review]
Look into these Points, else everything looks good to me.

File: Components/landingPage.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+}
+.card-price-list{
+    color: var(--color-primary);  
+    font-weight: bold;
+}
+.badge-label{
+    position: relative;
+    background:var(--color-link);
+    width: 5rem;
+    padding: 0.2rem;
```
Comment (Sahil25061999) [inline Components/landingPage.css]: Use px if value is less than 10

**Review by Sahil25061999 — APPROVED** [review]

File: Components/category.css (hunk 1 above)
Comment (Anurag021) [reply Components/category.css]: Thanks Done...

File: Components/landingPage.css (hunk 4 above)
Comment (Anurag021) [reply Components/landingPage.css]: Thanks Done...

File: Components/landingPage.css (hunk 3 above)
Comment (Anurag021) [reply Components/landingPage.css:20]: made all requested changes

**Pull request merged**